### PR TITLE
feat: add missing colors

### PR DESCRIPTION
**Pull request by laurentsenta**
- [x] add missing colors (grey-4, etc),
- [x] replace colors with a simple mixin to reduce code,
- [x] add label example with new colors.

### CODE REVIEW

**Review by amanharwara — APPROVED** [review]
Cool mixin!

File: stories/components/color.stories.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        options: [
+          "black",
+          "grey-1",
+          "grey-2",
+          "grey-3",
+          "grey-4",
+          "grey-5",
+          "grey-6",
+          "danger",
+          "success"
```
Comment (johnny243) [inline stories/components/color.stories.js]: This variant is `black` for me when I run Storybook (I think it doesn't exist)

File: stories/components/color.stories.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        ]
+      }
+    }
+  },
+};
+
+const Template = (args) => <label className={`color-${args.variant}`} >Helloworld</label>;
```
Comment (johnny243) [inline stories/components/color.stories.js]: Are we showing a `Button` (see line 6) or a `label`?

**Review by johnny243 — COMMENTED** [review]
🚀🚀🚀

**Review by vardan-arm — APPROVED** [review]

File: stories/components/color.stories.js (hunk 1 above)
Comment (laurentsenta) [reply stories/components/color.stories.js]: Nice catch, thanks for checking

File: stories/components/color.stories.js (hunk 2 above)
Comment (laurentsenta) [reply stories/components/color.stories.js]: It is a label, we're demo'ing styles here,
but there is no corresponding component you can use in code.

Are you familiar with storybook? (I'm not),
according to the doc the component parameter is required,
I'm not sure what's the right value to put there.

https://storybook.js.org/docs/react/api/csf#default-export

Comment (johnny243) [reply stories/components/color.stories.js]: > It is a label, we're demo'ing styles here, but there is no corresponding component you can use in code.

Then how about creating a component just for demo'ing? Could be something like:

```javascript
const DemoComponent = ({ variant, text }) => <label className={`color-${variant}`}>{text}</label>;
```

Then, you can pass it to the `component` parameter and create a template with it.

Comment (laurentsenta) [reply stories/components/color.stories.js]: Sounds good, I've used the DemoComponent from the story itself (don't want to burden the core with demo code)

**Pull request merged**